assets/files/etc/haproxy/haproxy.cfg.template

```diff
@@ -15,8 +13,6 @@ frontend  main
   bind :7443
   default_backend masters
 backend masters
-   option external-check
+   option httpchk GET /healthz HTTP/1.0
```

Comment:
Checked with this haproxy.cfg in my env, 
All members are DOWN (health check fail)